app/models/user.rb

```diff
@@ -4,4 +4,9 @@ class User < ApplicationRecord
 
   validates :name, presence: true
   validates :email, presence: true, uniqueness: true
+
+  #Look into refactoring (at some point)
```

Comment:
Agreed! We can either keep this here, or move to the Application Helper module, or adjust our controllers to accommodate the change - pending what we hear back from check-in tomorrow.